web-console/src/utils/druid-query.tsx

```diff
@@ -43,3 +43,60 @@ export async function queryDruidSql(sqlQuery: Record<string, any>): Promise<any[
   }
   return sqlResultResp.data;
 }
+
+function parseQueryPlanResult(queryPlanResult: string) {
+  if (!queryPlanResult) {
+    return {
+      query: null,
+      signature: null
+    };
+  }
+
+  const queryAndSignature = queryPlanResult.split(', signature=');
+  const queryValue = new RegExp(/query=(.+)/).exec(queryAndSignature[0]);
+  const signatureValue = queryAndSignature[1];
+
+  let parsedQuery: any;
+
+  if (queryValue && queryValue[1]) {
+    try {
+      parsedQuery = JSON.parse(queryValue[1]);
+    } catch (e) {}
+  }
+
+  return {
+    query: parsedQuery || queryPlanResult,
+    signature: signatureValue || null
+  };
+}
+
+export function parseQueryPlan(raw: string): any {
```

Comment:
Could you please explicitly write down the interface of the return type of this function?